README.md

```diff
@@ -17,6 +17,8 @@ my everyday use. It can be thought of as an alternate [porcelain][] for Git.
 Download the latest [release][releases] from GitHub.  Binaries are available for
 Linux and macOS.
 
+To use the version at `HEAD`, run `go get zombiezen.com/go/gg/cmd/gg`.
```

Comment:
I'd rather this link to the [Building from source](https://github.com/zombiezen/gg/blob/master/CONTRIBUTING.md#building-from-source) section, which might need to be updated. WDYT about cleaning up the instructions there?